site/en/blog/chips-origin-trial/index.md

```diff
@@ -22,17 +32,15 @@ This trial is available as a [third-party origin trial](/blog/third-party-origin
 
 If a site enrolls in the trial as a first-party, the cookie partitioning functionality will be available to any third-party content providers on that site as well. These third-party providers should also expect to receive extra HTTP headers, indicating their enrollment in the origin trial.
 
-##
-
 ## How long will the trial run?
 
-The trial will be available from Chrome 100 to Chrome 103. Check the [Chrome release schedule](https://chromiumdash.appspot.com/schedule) for the planned release dates.
+The trial will be available from Chrome 100 to Chrome 105. Check the [Chrome release schedule](https://chromiumdash.appspot.com/schedule) for the planned release dates.
 
 ## How to enroll in the origin trial
 
 ### Prerequisites
 
-Chrome stable 100.
+Chrome stable 103.
 
 ### Steps
```

Comment:
The `Origin-Trial` header should only be included in responses with `Set-Cookie` that include `Partitioned`, please update step 2.